web/profiles/custom/os2loop/modules/os2loop_upvote/src/Helper/FlagHelper.php

```diff
@@ -19,14 +19,16 @@ class FlagHelper {
   /**
    * Constructor.
    *
-   * @param \Drupal\flag\FlagCountManager $flag_service
-   *   The flag service.
+   * @param \Drupal\flag\FlagCountManager $flag_counter_manager
+   *   The flag count manager.
    */
-  public function __construct(FlagCountManager $flag_service) {
-    $this->flagService = $flag_service;
+  public function __construct(FlagCountManager $flag_counter_manager) {
```

Comment:
`$flag_counter_manager` should be renamed to `$flagCountManager` (camelCase) and `$flagService` in the class should also be renamed to this.